stubs/Pillow/PIL/MicImagePlugin.pyi

```diff
@@ -1,11 +1,15 @@
 from typing import Any, ClassVar
-from typing_extensions import Literal
+from typing_extensions import Literal, TypeAlias
 
 from .TiffImagePlugin import TiffImageFile
 
+_OleFileIO: TypeAlias = Any  # olefile.OleFileIO
+_OleStream: TypeAlias = Any  # olefile.OleStream
+
 class MicImageFile(TiffImageFile):
+    ole: _OleFileIO
     format: ClassVar[Literal["MIC"]]
-    fp: Any
-    frame: Any
-    def seek(self, frame) -> None: ...
-    def tell(self): ...
+    fp: _OleStream
+    frame: int
```

Comment:
Can be `None`: https://github.com/python-pillow/Pillow/blob/77b316c8f01bac2a7bf8b37a8010732d172ad6dc/src/PIL/MicImagePlugin.py#L67